### PR TITLE
chore: add suggestion for show image in submenu

### PR DESCRIPTION
**Pull request by habemuscode**
**User flow**

- User write `/start` and a list of dishes appears. It's just an `Inline Keyboard` with one button per dish
- User clicks on any of the buttons (pizza, for example). The previous `Inline Keyboard` is removed and appears a photo with some empty caption and another `Inline Keyboard` to manage paginations, add to favorites, or other actions

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
Can you resolve the merge conflicts before I review?

File: menu.ts [hunk 1]
```diff
@@ -57,6 +67,7 @@ const dishMenu = new Menu<MyContext>("dish");
 dishMenu.dynamic((ctx) => {
   const dish = ctx.match;
   if (typeof dish !== "string") throw new Error("No dish chosen!");
+  createMessageMedia(ctx, pizzaDatabase[0], createDishMenu(ctx.match))
```
Comment (KnorpelSenf) [inline menu.ts:71]: This is a side-effect. Whenever the menu is rendered, a message will be sent. This is not how the menu plugin is intended to be used.

File: menu.ts [hunk 2]
```diff
@@ -80,6 +91,15 @@ function createDishMenu(dish: string) {
     .row()
     .back({ text: "Back", payload: dish });
 }
+function createMessageMedia (ctx, product: Pizza | undefined, keyboard) {
+    const message = ctx.replyWithPhoto(
```
Comment (KnorpelSenf) [inline menu.ts:96]: You are not awaiting the promise, and the variable is unused.

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
I will push a commit to this branch that changes your code to how it should work.

File: menu.ts (hunk 2 above)
Comment (habemuscode) [reply menu.ts:96]: Yes, I forget to refactor this for the PR. In the original code (working without the menu plugin), I have `ctx.session.messageId = reply.message_id` to store this ID for future editing and also with the await, but I only remove the await. Thanks for the input

**Comment by KnorpelSenf** [comment]
I split up the changes in three commits. The middle one is the most important one.

- 947126a6a88f9290dcf6ca1b56f3c48607641158 only fixes formatting. It is not interesting.
- ed528dc253d207b9c709d7cf082619b229883405 applies the real changes, here you can see how it should work
- 2930a6cef69db64fc2a8cc1e1371776bdc8c5bbb does a lot of renaming from `dish` to `pizza`, and some minor changes. It creates a large diff, so I did not want to include it right away, but it is not really that interesting.

**Comment by KnorpelSenf** [comment]
You should maybe run the code and check if the user flow is what you're expecting it to be.

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
Please revert all changes to `menu.ts`. It is a good idea to add this example as an independent file. The current menu example should not be touched.

**Comment by habemuscode** [comment]
@KnorpelSenf I revert the changes but appear in diff because the `.editorconfig` (indent_size = 4) are applied and previously was committed without.

**Review by KnorpelSenf — APPROVED** [review]
Thank you for your work!

**Pull request merged**